Automattic/jetpack
# Social: Update the plan redirect to default to Advanced

We want to promote the advanced plan features, so instead of dropping people into the checkout for the basic plan, we should send people to the pricing page with the advanced plan selected.





## Proposed changes:

* This updates the redirect link to point to the Jetpack Pricing page with the Social Advanced plan selected.

### Other information:


## Jetpack product discussion


pdrWKz-YO-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:
* Activate the Jetpack Social plugin
* Ensure that you don't have a Social plan on the site
* On the Social admin page, check the link in the header to nudge you to upgrade sends you to the Jetpack pricing page with the information about the advanced plan displayed in the modal.
*
